datafusion/common/src/error.rs

```diff
@@ -778,6 +817,20 @@ mod test {
         );
     }
 
+    #[test]
+    fn test_strip_error_name() {
+        let res: Result<(), DataFusionError> = plan_err!("Err");
+        let res = res.unwrap_err();
+        assert_eq!(res.strip_error_name(), "Err");
+
+        // Test only top level stripped
+        let res: Result<(), DataFusionError> = plan_err!("Err");
```

Comment:
It looks like these are two scenarios that you are testing, should they go in two different test cases ?

Reply:
Thanks @edmondop I'd say the second scenario is variation of first one. Not sure if every variation requires its own test tbh